node/executor/src/lib.rs

```diff
@@ -895,6 +895,21 @@ mod tests {
 		assert!(t.storage_changes_root(GENESIS_HASH.into()).unwrap().is_some());
 	}
 
+	#[test]
+	fn should_import_block_with_test_client() {
```

Comment:
Do we currently support the opposite path? The real client using the real runtime? (recalling that this idea came along while I was trying to add a test with the real runtime in `core/basic-authorship`). I had a failed attempt so far.

Reply:
By using the new `generic-test-client` you should be able to do this.

Reply:
Yeah, despite the name `test-client` is actually The Real Client™ but with in-memory backend and (previously) test runtime. Now it's configurable and it's just a set of convenience methods to mock the client state.